docs/fundamentals/code-analysis/quality-rules/ca2234.md

```diff
@@ -41,15 +41,15 @@ To fix a violation of this rule, call the overload that takes the <xref:System.U
 
 It is safe to suppress a warning from this rule if the string parameter does not represent a URI.
 
-## Configurability
+## Configure code to analyze
 
-You can configure which parts of your codebase to run this rule on, based on their accessibility. For example, to specify that the rule should run only against the non-public API surface, add the following key-value pair to an *.editorconfig* file in your project:
+You can configure this option for just this rule, for all rules, or for all rules in this category (Usage). For more information, see [Code quality rule configuration options](../code-quality-rule-options.md).
 
-```ini
-dotnet_code_quality.ca2234.api_surface = private, internal
-```
+Use the following options to configure which parts of your codebase to run this rule on.
 
-You can configure this option for just this rule, for all rules, or for all rules in this category (Usage). For more information, see [Code quality rule configuration options](../code-quality-rule-options.md).
+- [Select API to analyze](#select-api-to-analyze)
```

Comment:
This hyperlink is exactly above the heading it's linking to. Feels redundant?

Reply:
That's what was decided for all the recent changes I made. Tagging @gewarren for advice.

Reply:
This is supposed to be an index. When there's only one option it isn't really necessary, but I think we should leave the index here in case more options are added in the future.